backend/tests/test_recovery_router.py

```
@@ -404,3 +406,160 @@ def test_recovery_plan_response_matches_schema(client):
+# ---------------------------------------------------------------------------
+# Issue #234 (W-H) — RecoveryOkrInputs V1 contract
+#
+# The five sizing-cap fields (sizing_cap_pct, total_capital,
+# resolved_sizing_cap_dollars, account_id_masked, capital_status) must
+# round-trip through the populated response, and the Sizing-cap assumption
+# row must render the §6.2 contract copy on both the ok path and the
+# cap-unenforceable path.
+# ---------------------------------------------------------------------------
+
+
+def _patch_account_cache(monkeypatch, result):
```

Ruff (0.15.13)

[warning] Missing return type annotation for private function `_patch_account_cache`

Add return type annotation: `None`

(ANN202)